### PR TITLE
[15.0][MIG] account_statement_import_online_paypal

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Supersedes https://github.com/OCA/bank-statement-import/pull/469.

### CODE REVIEW

File: account_statement_import_online_paypal/models/online_bank_statement_provider_paypal.py [hunk 1]
```diff
@@ -245,8 +249,12 @@ def _obtain_statement_data(self, date_since, date_until):
         )
         if not last_transaction:
             raise UserError(
-                _("Failed to resolve transaction %s (%s)")
-                % (last_transaction_id, last_transaction_date)
+                _(
+                    "Failed to resolve transaction %(last_transaction_id)s "
+                    "%(last_transaction_date)s",
```
Comment (astirpe) [inline account_statement_import_online_paypal/models/online_bank_statement_provider_paypal.py]: just noticed that `last_transaction_date` was in parenthesis, which seem removed with your changes, but this is not a big problem I guess

Comment (MiquelRForgeFlow) [reply account_statement_import_online_paypal/models/online_bank_statement_provider_paypal.py]: good catch, solved.

**Review by ghost — APPROVED** [review]

**Review by NL66278 — APPROVED** [review]
:+1: LGTM

**Comment by kwoychesko** [comment]
If this is approved I would love to have it merged. I will then create my PR which updates CONFIGURE and USAGE to match the latest Odoo menu layout and procedure for creating Journals and Bank Accounts, and the latest changes to PayPal's procedure for creating a Client ID/Secret (also how to configure this module to run against PayPal's Sandbox for testing/dev purposes).
Thanks!

**Comment by kwoychesko** [comment]
Thank you very much @pedrobaeza — much appreciated!

**Pull request merged**